editor/renames_map_3_to_4.cpp

```diff
@@ -1791,4 +1791,47 @@ const char *RenamesMap3To4::color_renames[][2] = {
 	{ nullptr, nullptr },
 };
 
+const char *RenamesMap3To4::theme_override_renames[][2] = {
+	// First rename the generic prefixes.
+	{ "custom_constants/", "theme_override_constants/" },
+	{ "custom_styles/", "theme_override_styles/" },
+	{ "custom_colors/", "theme_override_colors/" },
```

Comment:
```suggestion
	{ "custom_colors/", "theme_override_colors/" },
	{ "custom_constants/", "theme_override_constants/" },
	{ "custom_fonts/", "theme_override_fonts/" },
	{ "custom_icons/", "theme_override_icons/" },
	{ "custom_styles/", "theme_override_styles/" },
```

Unless there is a reason to avoid converting fonts and icons?

As a side note, Godot 4 projects also have font_sizes, but there is no equivalent in 3.x, so it's okay to skip. Godot 3 has custom_shaders, but those are deprecated, so also okay to skip.

Reply:
Icons are a good idea!
I'm not sure about fonts. Does that just work? Since Godot 4 just works with `.ttf` files and I figured my old font resources do not work anymore.

Reply:
Yes, old resources would need to be converted or remade, but this is true about a lot of things. StyleBoxes also don't necessarily work out of the box and need their own renames, for example. This converter is supposed to do its best effort, so I think it should try to rename all that is relevant the best it can.

Unless it crashes due to the font being an invalid resources, but that's a separate matter IMO.

Reply:
Okay, fine for me.
StyleBoxes actually work quite fine, I also tried more complex ones and they were just fine as well.